Engine/source/app/net/tcpObject.cpp

```diff
@@ -400,6 +405,29 @@ void TCPObject::send(const U8 *buffer, U32 len)
    Net::sendtoSocket(mTag, buffer, S32(len));
 }
 
+bool TCPObject::sendFile(const char* fileName)
+{
+	//Open the file for reading
+	FileStream readFile;
+	if(!readFile.open(fileName, Torque::FS::File::Read))
+	{
+		return false;
+	}
+
+	//Read each byte into our buffer
+	Vector<U8> buffer(readFile.getStreamSize());
+	U8 byte;
+	while(readFile.read(&byte))
+	{
+		buffer.push_back(byte);
+	}
+
+	//Send the buffer
+	send(buffer.address(), buffer.size());
```

Comment:
I remembered you submitted something similar to this James, but didn't have time to check if it was exactly this.